inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -185,6 +185,12 @@ cephfs_provisioner_enabled: false
 # cephfs_provisioner_secret: secret
 # cephfs_provisioner_storage_class: cephfs
 
+# Nginx ingress controller deployment
+nginx_ingress_enabled: false
+# nginx_ingress_namespace: "nginx-ingress"
```

Comment:
@Atoms and @woopstar so something like this?

Reply:
not just in group_vars but in defaults too.

Reply:
Yes default also updated, too

Reply:
default is not "nginx-ingress", but "ingress-nginx"

Reply:
It is a bit confusing me with "nginx-ingress" or "ingress-nginx"...

Officially, Kubernetes github project now coming with following 2 repos:
* https://github.com/kubernetes/ingress-gce
* https://github.com/kubernetes/ingress-nginx

So from this point of views, our integration should also using "ingress-nginx" as primary namespace, both for its internal components prefixing.

BTW, when reading the source code from https://github.com/kubernetes/ingress-nginx/tree/master/deploy, they keep using both "nginx-ingress" **AND** "ingress-nginx" together...

Anyway, from my point of view, we should keep our implementation in sync by using "ingress-nginx" **ONLY**.

Any idea?